scripts/www/build-cfg-help.pl

```diff
@@ -247,9 +247,12 @@ ()
     }
 }
 
+my $linecount = 0;
+my @ifelse = {};
```

Comment:
This is an array, not a hash:

```suggestion
my @ifelse = ();
```

This is one of the two problems insist on addressing.